Cartfile

```diff
@@ -2,5 +2,5 @@ binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.5
 github "mapbox/MapboxDirections.swift" ~> 0.10.0
 github "52inc/Pulley" ~> 1.3.1
 github "rs/SDWebImage" ~> 4.0
-github "Project-OSRM/osrm-text-instructions.swift" ~> 0.2.0
+github "Project-OSRM/osrm-text-instructions.swift" ~> 0.3
```

Comment:
We should pin any pre-1.0 dependency to a patch version (but still use the tadpole operator to allow for compatible upgrades). Fixed in 25af72698ef0323591aeb0cc99d42604f9ad67f5.